docs/userguide/networking/overlay-security-model.md

```diff
@@ -0,0 +1,62 @@
+<!--[metadata]>
```

Comment:
@diogomonica @nathanleclaire agreed. Yes. we must introduce the concept of encrypted data-path pointing to `-o encrypted` flag in `docker network create` command.